src/features/emails/l10n/messageIds.ts

```diff
@@ -63,6 +63,7 @@ export default makeMessages('feat.emails', {
           noButtonText: m('Click to change this text!'),
         },
         settings: {
+          defaultButtonTextWarning: m('This button still has the default text'),
```

Comment:
Maybe we can change this to something that works regardless of what the error is? Something like "Make sure to add your text to the button" or something like that?

I trust you to come up with something reasonable. Just wanted to propose a bit of a shortcut here that might work. 😊